client/pages/projects/[projectId]/index.jsx

```diff
@@ -52,8 +107,16 @@ export default function ProjectPage() {
           setProject(model.data);
         }
       }
+      setLoading(true);
+      const resp = await getFollowingProjects();
+      const res = await resp.json();
+      if (projectId && res.data.includes(projectId)) {
```

Comment:
i'm getting a frontend error on this line in the deployed version
i dont think it has to do with my browser but try navigating to this page on a deployed app and see what happens